core/resources/scene/config.yaml

```diff
@@ -16,6 +16,12 @@ textures:
     water-texture:
         url: normals.jpg
         filtering: nearest
+
+# provide global fontfile names to be used for this stylesheet
+# these could be bundled with the tangram repo or could be loaded from system path
+# priority is given to bundled fonts
+fonts: [ Roboto-Regular.ttf, FiraSans-Medium.ttf]
```

Comment:
Good point.
We can have a list of strings something respecting the following format:
```
"FontName, Weight, Style, Extension"
```
Which can then be parsed as (for android): FontName-WeightStyle.ext